jupyter_book/commands/__init__.py

```diff
@@ -108,7 +108,10 @@ def build(path_book, path_output, config, toc, warningiserror, build):
                 {path_output_rel}{os.sep}
 
             You can look at your book by opening this file in a browser:
-                {path_index}\
+                {path_index}
+
+            Or paste this line directly into your browser bar:
+                file://{OUTPUT_PATH}\
```

Comment:
I think this needs to be:

```suggestion
                file://{path_index.resolve()}\
```

so that it's pointing to an actual file instead of a folder